play/PlayerRules/Basic Rules.txt

```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
+shoot more than 1 bullet per Action by default).
 
 Gun Jams:
-On a roll of 00-01, the weapon Jams, necessitating a weapon skill check, DC 60.
-When a player is defending against a hit, they roll Percentiles under their 
-armor in order to apply a Damage Reduction, or to apply damage to their 
-Capacitive Sheilds if the player has them.
+If at any time that you are firing a gun and you roll a weapon accuracy of 1, roll a second d10.
+If that second d10 roll is a 1 or 2, the weapon Jams, necessitating a weapon 
+skill check, DC 60. 
 
-Hip Firing doubles a weapon''s miss chance. It is much smarter to takes one 
-action to Aim, and spend the rest of your shots on firing. Its also a great 
-idea to take an action at the end of your turn to "hunker," and gain a 25% 
+Armor and Getting Hit:
+When a player is defending against a hit, they roll Percentiles under their 
+armor in order to see if their armor blocked the bullet, and from there, how much
+damage they receive. If the player has no armor, they roll for a target 50 Percentile. 
+If the shot/attack rolls above 50%, they receive half-damage. Below 50%, full damage.
+
+Armor consists of 3 concepts, Armor Points, Armor Piercing (AP) level, and % Coverage.
+Starting from the end, Coverage is simply the percentile that you have to roll
+under to see if armor was hit. If the attack misses armor, or rolls over 50%, it does
+half damage. If the attack hits armor: AP level comes into play. 
+	AP level is a unitless number that represents how well a weapon can pierce armor or
+how well armor can block bullets. If the attacker's weapon AP level is less than the
+defender's armor AP level, all damage is blocked. If the weapon AP level is greater than
+or equal to the armor AP level, then Armor Points come into play. 
+	Armor Points are, effectively, your armor's ability to slow down projectiles as they
+pass through it. Whenever armor is successfully pierced, all damage is first inflicted
+on Armor Points. Once the armor points are used up, all remaining damage goes to health.
+Armor points refresh at the start of the armor wearer's turn.
+
+Hip Firing:
+Add 3 to a weapon''s miss chance. You cannot use attachments to assist in
+hip firing unless otherwise noted. It is much smarter to takes one 
+action to Aim, and spend the rest of your shots firing. It's also a great 
+idea to take an action at the end of your turn to "hunker," and gain a +2 
 cover advantage.
+If your weapon has a minimum distance (usually because of a high-powered
+scope), and you fire inside that distance, you are forced to hip fire.
+
+Blind Firing:
+Add 6 to a weapon's miss chance. Blind firing allows you to shoot around cover
+without looking or putting yourself at risk. Your blind fire accuracy is affected 
+by your base weapon miss chance(s), not affected by regular attachments, and only 
+reduced by LUCK and PER mod (max of -4 miss chance from LUCK mod + PER combat mod).
 

```

Comment:
I also like that blindfiring is available to everybody. Please adapt the blindfire feat acordingly or assign that work to another dev and have them merge that to the superbranch.